_posts/2015-09-03-sqlalchemy-and-django.md

```diff
@@ -3,7 +3,7 @@ layout: post
 title: SQLAlchemy and Django
 author: Rebecca
 ---
-At [BetterWorks](http://betterworks.com), we use the [Django ORM](https://docs.djangoproject.com/en/1.8/topics/db) for data persistence and have found it to be very convenient for modeling and basic querying. It's very easy to use and does a lot of the work for you when it comes to dealing with multiple tables in the database. However, it falls short for some of the more complex queries.
+At [BetterWorks](https://betterworks.com), we use the [Django ORM](https://docs.djangoproject.com/en/1.8/topics/db) for data persistence and have found it to be very convenient for modeling and basic querying. It's very easy to use and does a lot of the work for you when it comes to dealing with multiple tables in the database. However, it falls short for some of the more complex queries.
```

Comment:
the http vs https isn't super necessary because we auto-redirect the users to https